pkg/skaffold/schema/latest/config.go

```diff
@@ -35,13 +35,22 @@ type SkaffoldConfig struct {
 	// Kind is always `Config`. Defaults to `Config`.
 	Kind string `yaml:"kind" yamltags:"required"`
 
+	// Metadata holds additional information about the config.
+	Metadata Metadata `yaml:"metadata,omitempty"`
+
 	// Pipeline defines the Build/Test/Deploy phases.
 	Pipeline `yaml:",inline"`
 
 	// Profiles *beta* can override be used to `build`, `test` or `deploy` configuration.
 	Profiles []Profile `yaml:"profiles,omitempty"`
 }
 
+// Metadata holds an optional name of the project.
+type Metadata struct {
+	// Name is an identifier for the project.
```

Comment:
@balopat Would this be a good place to warn about non-unique project names?

Reply:
Maybe, as we implement the kubecontext switching. It wouldn't make sense before it.